Security/Exception/LtiToolMessageExceptionHandler.php

```diff
@@ -67,4 +67,14 @@ public function handle(Throwable $exception, Request $request): Response
 
         throw $exception;
     }
+
+    private function getTokenIdFrom(Request $request): string
```

Comment:
you're doing those things just for psalm ?

Reply:
I like the idea of ​​renaming the method.

Moving to a new class for now is not necessary, in this case it will return a string and in another case maybe return an int.

Reply:
afaik in LTI every parameters during OIDC flow are strings